functions/package.json

```diff
@@ -12,6 +12,7 @@
     "firebase": "^8.2.4",
     "firebase-admin": "^9.4.2",
     "firebase-functions": "^3.13.1",
+    "lodash": "^4.17.21",
```

Comment:
why add lodash?

Reply:
needed here
https://github.com/sparkletown/sparkle/blob/d9e007ad7934d1fb203f4dabfaffbbbe87692339/functions/auth.js#L16

Reply:
Not really representative explanation "library is needed so that we can require a function from it"

Reply:
So should I try to chunk the Users by a loop?